examples/README.md

```diff
@@ -1,76 +1,78 @@
 # Example
 
```

Comment:
Please consider adding a few words what this example create

Reply:
done